lib/init.coffee

```diff
@@ -9,42 +11,83 @@ module.exports =
       type: 'string'
       title: 'Custom rules directory'
       default: ''
+    useLocalTslint:
+      type: 'boolean'
+      title: 'Try using the local tslint package (if exist)'
+      default: true
+
+  rulesDirectory: ''
+  tslintCache: new Map
+  tslintDef: null
+  useLocalTslint: true
```

Comment:
No need for this, the `observe` in activate will set it for you when the plugin is activated.